packages/playground/src/utils/validators.ts

```diff
@@ -777,3 +777,11 @@ export async function isAvailableName(grid: GridClient, name: string) {
     return { message: "Name is already taken." };
   }
 }
+
+export function validateSmtp(v: string) {
```

Comment:
we may name it `IsValidSmtp` or something similar, also, `v` to be something like `input` to increase readability

Reply:
I'm fine about naming but for ```v``` it stands for ```value``` whatever it looks like, like the rest of the validators